editions/tw5.com/tiddlers/macros/TableOfContentsMacro.tid

```diff
@@ -62,4 +62,38 @@ These two parameters are combined into a single [[filter expression|Filter Expre
 ;template
 : Optionally, the title of a tiddler to use as a [[template|TemplateTiddlers]] for transcluding the selected tiddler into the right-hand panel
 
-[[Examples|Table-of-Contents Macros (Examples)]]
+!! Custom Icons
+
+<<.from-version "5.2.4">>
+
+It is possible to define the `toc-open-icon` and `toc-closed-icon` macros or variables, to change the icons used by the toc macros. This setting works for all toc-macro variants.
+
+!!! Default Settings
+
+* <<.var toc-open-icon>>: `\define toc-open-icon() $:/core/images/down-arrow`  
+
+* <<.var toc-closed-icon>>: `\define toc-closed-icon() $:/core/images/right-arrow` 
+
+!!! Custom Definitions
+
+The default settings can be overwritten in your code using the define-pragma or the let-widget. The let-widget can be used, if you have multiple toc-macros in one tiddler, that need different icons. 
```

Comment:
The best way to link to a widget is `<<.wlink LetWidget>>` (although note that you'll need to include the word "widget" after the macro call).

Also we don't say "toc-macros", we spell it out as "table of contents".